mesheryctl/pkg/utils/healthcheck_test.go

```diff
@@ -95,3 +97,88 @@ func TestCheckKubectlVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckK8sVersion(t *testing.T) {
+	tests := []struct {
+		version *version.Info
+		expected [3]int
+	}{
+		{
+			version: &version.Info{
+				Major:        "1",
+				Minor:        "2",
+				GitVersion:   "v1.12.0",
+				GitCommit:    "abcdefg",
+				GitTreeState: "clean",
+				BuildDate:    "2023-06-20",
+				GoVersion:    "go1.16",
+				Compiler:     "gc",
+				Platform:     "linux/amd64",
+			},
+			expected: [3]int{1, 12, 0},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run("Check K8s Version", func(t *testing.T) {
+			if got := CheckK8sVersion(tt.version); got != nil {
+				t.Errorf("getK8sVersion() = %v, want %v", got, tt.expected)
+			}
+		})
+	}
+
+}
+
+
+func TestIsCompatibleVersion(t *testing.T){
+	tests := []struct {
+		minimum [3]int
+		current [3]int
+		expected bool
+	}{
+		{
+			minimum: [3]int{1, 12, 0},
+			current: [3]int{1, 12, 0},
+			expected: true,
+		},
+		{
+			minimum: [3]int{1, 11, 0},
+			current: [3]int{2, 13, 2},
+			expected: true,
+		},
```

Comment:
can we have one negative test case here as well?
i.e. current < minimum version?

Reply:
sure, I just added it